recipes-devtools/rust/rust_1.12.1.bb

```diff
@@ -0,0 +1,26 @@
+inherit rust-installer
+require rust.inc
+require rust-source-${PV}.inc
+
+DEPENDS += "cmake-native"
```

Comment:
No. The configure script checks for cmake and doesn't allow the configure script to succeed without it. Approximately line 850.

```
# For building LLVM
probe_need CFG_CMAKE cmake
```